lib/assets/stylesheets/chef/components/form.scss

```diff
@@ -8,34 +8,30 @@ category: Form
 ## Sign-In Form
 
 ```html_example
-<form data-abide>
-  <div class="row">
-    <div class="medium-6 columns username-field">
-      <label>Username
-        <input type="text" required />
-      </label>
-      <small class="error">Required</small>
-    </div>
-  </div>
-  <div class="row">
-    <div class="medium-6 columns password-field">
-      <label>Password
-        <input type="password" required />
-      </label>
-      <small class="error">Required</small>
-    </div>
-  </div>
-  <div class="row">
-    <div class="medium-6 columns">
-      <button type="submit" class="button">Sign In</button>
-    </div>
-  </div>
-  <div class="row">
-    <div class="medium-6 columns">
-      <a href="#">Forgot your password?</a>
-    </div>
+<div class="row">
+  <div class="medium-6 columns">
+    <form>
+      <div>
+        <label>
+          Username
+          <input type="text" class="radius">
```

Comment:
We should use HTML5 validation. Put `required` on this and the password. We probably could do a `pattern` as well if you want to go find in the Erlang where those are defined for username and password.